demos/projects/ESPRESSIF/aziotkit/config/demo_config.h

```diff
@@ -0,0 +1,172 @@
+ */
+
+#include "esp_log.h"
+
+#ifndef LIBRARY_LOG_NAME
+    #define LIBRARY_LOG_NAME    "AzureIoTDemo"
+#endif
+
+#define SINGLE_PARENTHESIS_LOGE( x, ... ) ESP_LOGE( LIBRARY_LOG_NAME, x, ##__VA_ARGS__ )
+#define LogError( message )               SINGLE_PARENTHESIS_LOGE message
+
+#define SINGLE_PARENTHESIS_LOGI( x, ... ) ESP_LOGI( LIBRARY_LOG_NAME, x, ##__VA_ARGS__ )
+#define LogInfo( message )                SINGLE_PARENTHESIS_LOGI message
+
+#define SINGLE_PARENTHESIS_LOGW( x, ... ) ESP_LOGW( LIBRARY_LOG_NAME, x, ##__VA_ARGS__ )
+#define LogWarn( message )                SINGLE_PARENTHESIS_LOGW message
+
+#define SINGLE_PARENTHESIS_LOGD( x, ... ) ESP_LOGD( LIBRARY_LOG_NAME, x, ##__VA_ARGS__ )
+#define LogDebug( message )               SINGLE_PARENTHESIS_LOGD message
+
+/************ End of logging configuration ****************/
+
+    /**
+     * @brief The model id for this device
+     */
+#define sampleazureiotMODEL_ID                                "dtmi:espressifEsp32AzureIotKit:EspressifESP32AzureIoTKit5x3;1"
+
+
+/**
+ * @brief Enable Device Provisioning
+ *
+ * @note To disable Device Provisioning undef this macro
+ *
+ */
+
+#ifdef CONFIG_ENABLE_DPS_SAMPLE
+    #define democonfigENABLE_DPS_SAMPLE
+#endif
+
+#ifdef democonfigENABLE_DPS_SAMPLE
+
+/**
+ * @brief Provisioning service endpoint.
+ *
+ * @note https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#service-operations-endpoint
+ *
+ */
+#define democonfigENDPOINT "global.azure-devices-provisioning.net"
+
+/**
+ * @brief Id scope of provisioning service.
+ *
+ * @note https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#id-scope
+ *
+ */
+#define democonfigID_SCOPE CONFIG_AZURE_DPS_ID_SCOPE
+
+/**
+ * @brief Registration Id of provisioning service
+ *
+ * @warning If using X509 authentication, this MUST match the Common Name of the cert.
+ *
+ *  @note https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#registration-id
+ */
+#define democonfigREGISTRATION_ID CONFIG_AZURE_DPS_REGISTRATION_ID
+
+
+#endif // democonfigENABLE_DPS_SAMPLE
+
+/**
+ * @brief IoTHub device Id.
+ *
+ */
+#define democonfigDEVICE_ID CONFIG_AZURE_IOT_DEVICE_ID
+
+/**
+ * @brief IoTHub module Id.
+ *
+ * @note This is optional argument for IoTHub
+ */
+#define democonfigMODULE_ID CONFIG_AZURE_IOT_MODULE_ID
+
+/**
+ * @brief IoTHub hostname.
+ *
+ */
+#define democonfigHOSTNAME CONFIG_AZURE_IOT_HUB_FQDN
+
+/**
+ * @brief Device symmetric key
+ *
+ */
+#define democonfigDEVICE_SYMMETRIC_KEY CONFIG_AZURE_IOT_DEVICE_SYMMETRIC_KEY
+
+/**
+ * @brief Client's X509 Certificate.
+ *
+ */
+// #define democonfigCLIENT_CERTIFICATE_PEM    "<YOUR DEVICE CERT HERE>"
+
+/**
+ * @brief Client's private key.
+ *
+ */
+// #define democonfigCLIENT_PRIVATE_KEY_PEM    "<YOUR DEVICE PRIVATE KEY HERE>"
+
+/**
+ * @brief Baltimore Trusted Root CA.
+ *
+ */
+#define democonfigROOT_CA_PEM "-----BEGIN CERTIFICATE-----\r\n" \

```

Comment:
can we add the new DigiCert Global G2 CA to this as well?

Reply:
We will do that as an improvement to the samples as a whole, in a separate PR.
To double-confirm, the other existing samples also do not have the new server root cert, so all samples need to be updated.
I'll keep this comment unresolved for tracking.

Reply:
You can close this. I created the Issue #99 to track verifying and fixing the certs on samples.